tests/tensor_search/test_media_download_thread_count.py

```diff
@@ -0,0 +1,175 @@
+import unittest
```

Comment:
These don't seem relevant to the PR

Reply:
Thanks, yeah they're unrelated but I added for extra coverage. I've now deleted this file and created another PR to add it: https://github.com/marqo-ai/marqo/pull/1019